### PR TITLE
render workflow changes which helps to delete pr DB

### PR DESCRIPTION
**Pull request by adishM98**
added the code to delete the db when destroy label is triggered

### CODE REVIEW

File: .github/workflows/render-preview-deploy.yml [hunk 1]
```diff
@@ -242,6 +242,16 @@ jobs:
...
 
+      - name: Drop PostgreSQL PR database
+        env:
+          PGHOST: dpg-cf4metg2i3mtiflq3dvg-a.oregon-postgres.render.com
+          PGPORT: 5432
+          PGUSER: preview_env_db_user
+          PGPASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
+          PGDATABASE: ${{ env.PR_NUMBER }}
+        run: |
+          psql -h $PGHOST -p $PGPORT -U $PGUSER -d postgres -c "DROP DATABASE $PGDATABASE;"
```
Comment (akshaysasidrn) [inline .github/workflows/render-preview-deploy.yml:253]: can we infer this from `${{ secrets.RENDER_PG_HOST }}`

File: .github/workflows/render-preview-deploy.yml [hunk 2]
```diff
@@ -242,6 +242,16 @@ jobs:
               console.log(e)
             }
 
+      - name: Drop PostgreSQL PR database
+        env:
+          PGHOST: dpg-cf4metg2i3mtiflq3dvg-a.oregon-postgres.render.com
+          PGPORT: 5432
+          PGUSER: preview_env_db_user
+          PGPASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
```
Comment (akshaysasidrn) [inline .github/workflows/render-preview-deploy.yml:250]: ${{ secrets.RENDER_PG_PASS }}

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: .github/workflows/render-preview-deploy.yml [hunk 3]
```diff
@@ -242,6 +242,16 @@ jobs:
               console.log(e)
             }
 
+      - name: Drop PostgreSQL PR database
```
Comment (akshaysasidrn) [inline .github/workflows/render-preview-deploy.yml:245]: Also I think we should remove the label after deleting the db

Comment (adishM98) [reply .github/workflows/render-preview-deploy.yml:245]: Since i have added it under the label destroy-review-app it will be removed automatically.

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**